pkgs/tools/security/vault-ssh-helper/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, lib, fetchFromGitHub, buildGoModule, installShellFiles, nixosTests
+, makeWrapper
+, gawk
+, glibc
+}:
+
+buildGoModule rec {
+  pname = "vault-ssh-helper";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "hashicorp";
+    repo = "vault-ssh-helper";
+    rev = "v${version}";
+    sha256 = "sha256-TY+0EqQW2vUh6JIDHI+98leOwBT9ZhsVTJL0FpZrXjo=";
+  };
+
+  vendorHash = null;
+
+  nativeBuildInputs = [ installShellFiles makeWrapper ];
+
+  tags = [ "vault-ssh-helper" ];
+
+  ldflags = [
+    "-s" "-w"
+    "-X github.com/hashicorp/vault-ssh-helper/sdk/version.GitCommit=${src.rev}"
```

Comment:
`version.GitCommit` is only used when `version.VersionPrerelease != ""`.
Just a note, we can still keep it.

https://github.com/hashicorp/vault-ssh-helper/blob/v0.2.1/version.go#L20-L26